src/i18n/en-US/index.ts

```diff
@@ -190,6 +190,7 @@ export default {
         'If you already have an account ID that is created on Polkadot.js app, this account unification information will override.',
       agreeToProceed:
         'I have read all and I would like to proceed to create a unified account please.',
+      agreeToSubmit: 'I understand this action will not be canceled.',
```

Comment:
Maybe to provide a bit more explanation, something like to add: Once accounts are unified they can't be separated. 
Probably need a better word than separate :)

Reply:
Maybe `I understand that once accounts have been unified, they cannot be separated.` ?  @gluneau @Kahonnohak @ayumitk Please advise 🙏

Reply:
<img width="516" alt="image" src="https://github.com/AstarNetwork/astar-apps/assets/92044428/73e6de25-56a9-4fcb-a72e-20730eb585ce">

Reply:
> Maybe `I understand that once accounts have been unified, they cannot be separated.` ? @gluneau @Kahonnohak @ayumitk Please advise 🙏

happy with it!